ApiCustomer/AppServices/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace AppServices.Extensions;
+
+public static class DateTimeExtensions
+{
+    public static bool BeOver18(this DateTime dataNascimento)
+    {
+        if (DateTime.Now.Year - dataNascimento.Year >= 18)
+        {
+            return true;
+        }
+
+        return false;
```

Comment:
Essa lógica não funciona, se quiser entregar esse "extra" tem que corrigir a lógica
```suggestion
        return DateTime.Now.Year - dataNascimento.Year >= 18;
```